app/views/shared/_header_menu.html.erb

```diff
@@ -0,0 +1,6 @@
+<ul class="nav nav-background nav-txt-color p-2">
```

Comment:
Given that we have only one list item inside here... what do you think of using a `nav` tag instead of `ul`? 🤔 
That will improve tremendously the semantic relevance of the markup!
We can even keep this `ul > li` estructure, but housed inside a `nav` 😉